src/react-native/20-building/building.md

```diff
@@ -10,17 +10,40 @@
 
 In this section, you will:
 
-- TODO
+- Cover EAS production builds
+- Learn how to create a build
+- Understand the difference between APK and AAB
+- Learn about signing
 
 ## Objective 1: Creating a build
 
-TODO
+### EAS Production Build
 
-### Concept TODO
+EAS, or Expo Application Services, is a service provided by Expo to help streamline the process of building and deploying React Native applications. It is a cloud-based solution that allows us to create production builds, submit them to the app stores, and manage OTA (over-the-air) updates.
```

Comment:
I would move this to the very end. I’m thinking about cutting it altogether (I don’t think it fits with our content).

Reply:
moved